### PR TITLE
Testing dust prior weights

### PR DESCRIPTION
**Pull request by mdecleir**
Added a script that tests the functionality in "prior_dust_weights.py". This is part of issue #520

### CODE REVIEW

File: beast/physicsmodel/tests/test_dust_prior_weights.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
+
+#from beast.physicsmodel.prior_weights_dust import set_av_weights
```
Comment (karllark) [inline beast/physicsmodel/tests/test_dust_prior_weights.py:4]: Please remove any commented out code.  Best practice.

File: beast/physicsmodel/tests/test_dust_prior_weights.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    rv_vals = [2., 3., 4., 5., 6.]
+    fA_vals = [0.0, 0.2, 0.4, 0.6, 0.8, 1.0]
+    av_model = {"name":"flat"}
+    rv_model = {"name":"flat"}
+    fA_model = {"name":"flat"}
+
+    prior_weights = PriorWeightsDust(av_vals, av_model, rv_vals, rv_model, fA_vals, fA_model)
+
+    # test if priors are set correctly when creating an object of the class PriorWeightsDust
+    np.testing.assert_allclose(prior_weights.av_priors,np.full((len(av_vals)), 1.0))
```
Comment (karllark) [inline beast/physicsmodel/tests/test_dust_prior_weights.py:22]: Would be great to add an error message (using the err_msg keyword) to make it a little easier to know what happened when the assert fails. 
Same comment for all the assert statements.

Comment (mdecleir) [reply beast/physicsmodel/tests/test_dust_prior_weights.py]: Done. Are these messages what you had in mind?

File: beast/physicsmodel/tests/test_dust_prior_weights.py [hunk 3]
```diff
@@ -19,28 +18,28 @@ def test_dust_prior_weights():
...
     # test other models (with random input values)
     expected_lognormal_av = [0., 2.00684868, 1.57828555, 1.09756182, 0.7677122, 0.54959176]
     prior_weights.set_av_weights(model={"name" : "lognormal", "max_pos" : 2.0, "sigma" : 1.0})
-    np.testing.assert_allclose(prior_weights.av_priors, expected_lognormal_av)
+    np.testing.assert_allclose(prior_weights.av_priors, expected_lognormal_av, err_msg="A problem occurred while setting the A(V) priors to lognormal.")
 
     expected_two_lognormal_av = [0.0, 5.962474, 3.07642414e-02, 4.46347150e-03, 1.60916384e-03, 6.89125670e-04]
     prior_weights.set_av_weights(model={"name" : "two_lognormal", "max_pos1" : 0.2, "max_pos2": 2.0, "sigma1" : 1.0, "sigma2" : 0.2, "N1_to_N2" : 1.0 / 5.0})
-    np.testing.assert_allclose(prior_weights.av_priors, expected_two_lognormal_av)
+    np.testing.assert_allclose(prior_weights.av_priors, expected_two_lognormal_av, err_msg="A problem occurred while setting the A(V) priors to two_lognormal."))
```
Comment (karllark) [inline beast/physicsmodel/tests/test_dust_prior_weights.py:38]: There is an extra `)` at the end of this line.  That is causing the tests to fail.  Same issue for at least a couple of the other similar lines.

Comment (mdecleir) [reply beast/physicsmodel/tests/test_dust_prior_weights.py]: Argh, stupid mistake! Sorry about that :)

**Pull request merged**